tasks/sign-binaries/tests/test-sign-binaries-checksum.yaml

```diff
@@ -0,0 +1,169 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-sign-binaries-checksum
+spec:
+  description: |
+    Run the sign-binaries task and verify the results
+  workspaces:
+    - name: tests-workspace
+  tasks:
+    - name: setup
+      workspaces:
+        - name: data
+          workspace: tests-workspace
+      taskSpec:
+        workspaces:
+          - name: data
+        steps:
+          - name: setup-values
+            image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+            script: |
+              #!/usr/bin/env sh
+              set -eux
+               cat > "$(workspaces.data.path)/data.json" << EOF
+              {
+                "somekey": {
+                  "key": "value",
+                  "foo": "bar"
+                }
+              }
+              EOF
+              mkdir -p "$(workspaces.data.path)/content/linux"
+              cat > "$(workspaces.data.path)/content/linux/linux_binary.bin" << EOF
+              some data
+              EOF
+
+              # Create release.json with spec author
+              cat > "$(workspaces.data.path)/release.json" << EOF
+              {
+                "spec": {
+                  "author": "John Doe"
```

Comment:
author is not in the release spec so this is not really a good example https://github.com/konflux-ci/release-service/blob/main/api/v1alpha1/release_types.go